contracts/multisig/src/key.rs

```diff
@@ -245,6 +244,8 @@ fn validate_and_normalize_public_key(
         .as_ref()
         .into()),
         // TODO: verify encoding scheme is standard
```

Comment:
remove this todo?

Reply:
this TODO corresponds to another task: https://axelarnetwork.atlassian.net/browse/AXE-3313. cc @milapsheth do we still need this?